Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "wildmatch"
-version = "2.1.1-alpha.0"
+version = "2.2.1-alpha.0"
```

Comment:
Please leave the version info. It will automatically be adjusted by my release script.